launch/turtlebot3_navigation.launch.py

```diff
@@ -0,0 +1,52 @@
+# /*******************************************************************************
+# * Copyright 2019 ROBOTIS CO., LTD.
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *     http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *******************************************************************************/
+
+# /* Author: Darby Lim */
+
+import os
+from ament_index_python.packages import get_package_share_directory
+import launch
+from launch import LaunchDescription
+from launch.actions import IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration
+
+TURTLEBOT3_MODEL = os.environ['TURTLEBOT3_MODEL']
+MAP_PATH = os.environ['HOME']
+
+def generate_launch_description():
+    use_sim_time = launch.substitutions.LaunchConfiguration('use_sim_time', default='false')
+    print('path', os.path.join(
+                      MAP_PATH,
+                      'map.yaml'))
+    map_path = os.path.join(
+                    os.environ['HOME'],
+                    'map.yaml')
+    map_dir = LaunchConfiguration(
+                    'map',
+                    default=map_path)
+    person_detection_launch_dir = get_package_share_directory('turtlebot3_description_reduced_mesh')
```

Comment:
This variable name shouldn't say `person_detection_*`.

Reply:
Eventually, we'll want to obtain the `turtlebot3_description_reduced_mesh` package from `git@github.com:aws-robotics/turtlebot3-description-reduced-mesh.git`, instead of from a subfolder under `aws-robomaker-sample-application-persondetection`.

Reply:
1. Changed the variable name which doesnt include person_detection_*.
2. Providing param through turtlebot3_navigation.launch.py required for initial_pose in the navigation2.launch.py. "param consists of waffle_pi.yaml (application specific) which is placed in person_detection_simulation subfolder in the aws-robomaker-sample-application-persondetection" .

Reply:
For 2., if this initial pose can't be found in the `waffle_pi.yaml` of the `turtlebot3_bringup`, `turtlebot3_navigation2`, or `turtlebot3_node` packages, I would prefer you move the `waffle_pi.yaml` that you need from the `person_detection_simulation` package to here under `aws-robomaker-small-house-world`.

Reply:
Yes, moved param folder to `aws-robomaker-small-house-world ` .

Reply:
Thanks!